pkg/cache/task.go

```diff
@@ -415,3 +416,21 @@ func (task *Task) enterState(event *fsm.Event) {
 		zap.String("destination", event.Dst),
 		zap.String("event", event.Event))
 }
+
+func PublishTaskEvent(taskID, reason, msg string, app interfaces.ManagedApp) error {
+	log.Logger.Info("taskID", zap.String("taskID", taskID))
```

Comment:
I am in double mind here.
1. taskID alone is not enough here, We may need more context here for helping to debug later.
2. May be its a debug log itself?

Reply:
I used for debugging this patch, but it makes sense to print this information in debug level.

Reply:
Actually a bit later, we already have a debug log. I added a more meaningful message to that.